cmd/metal-api/internal/service/partition-service.go

```diff
@@ -207,6 +209,26 @@ func (r *partitionResource) createPartition(request *restful.Request, response *
 		commandLine = *requestPayload.PartitionBootConfiguration.CommandLine
 	}
 
+	var waitingpoolminsize string
+	if requestPayload.PartitionWaitingPoolMinSize != nil {
+		waitingpoolminsize = *requestPayload.PartitionWaitingPoolMinSize
+	} else {
+		waitingpoolminsize = "0%"
```

Comment:
Allow both to be zero would be cleaner I think because existing database entries are then also "valid" and we do not need a migration. If you consider the `ScalerRange` type as proposed earlier on, you could also add `IsDisabled` and then skip initializing the PoolScaler before passing it to the FSM.